docker/mainnet/connector/connector.mainnet.conf

```diff
@@ -0,0 +1,54 @@
+[Application Options]
+datadir=/root/.connector
+debuglevel=debug
+network=mainnet
+# RPC address to bind
+rpchost=0.0.0.0
+rpcport=9002
+# exchange address
+enginehost=mainnet.exchange.bitlum.io
+engineport=8080
+
+[Bitcoin]
+bitcoin.minconfirmations=1
+bitcoin.syncdelay=5
+bitcoin.host=bitcoin.mainnet
+bitcoin.port=8332
+bitcoin.feeperunit=87
+
+[Bitcoincash]
+bitcoincash.minconfirmations=1
+bitcoincash.syncdelay=5
+bitcoincash.host=bitcoin-cash.mainnet
+bitcoincash.port=9332
+bitcoincash.feeperunit=350
+
+[Dash]
+dash.minconfirmations=1
+dash.syncdelay=5
+dash.host=dash.mainnet
+dash.port=10332
+dash.feeperunit=350
+
+[Ethereum]
+ethereum.minconfirmations=1
+ethereum.syncdelay=5
+ethereum.host=ethereum.mainnet
+ethereum.port=11332
+
+[Litecoin]
+litecoin.minconfirmations=1
+litecoin.syncdelay=5
+litecoin.host=litecoin.mainnet
+litecoin.port=12332
+litecoin.feeperunit=350
+
+[Bitcoinlightning]
+bitcoinlightning.tlscertpath=/root/.lnd/tls.cert
+bitcoinlightning.macaroonpath=/root/.lnd/admin.macaroon
+# lnd RPC address
+bitcoinlightning.host=bitcoin-lightning.mainnet
+bitcoinlightning.port=10009
+# lnd P2P address
+bitcoinlightning.peerhost=connector.bitlum.io
```

Comment:
Do we have this domain name created?

Reply:
Not yet.

Reply:
Will add this domain when will start to test mainnet.